### DISCUSSION TITLE
Github Sheet Cheat

### DISCUSSION THREAD

**Discussion by Ayuyon** [community/community#69025] [New to GitHub]
Hi everyone! 

Does someone have a github sheet cheat to share?

**Comment by xtycoonfi** [comment]
Hey! 

I have found this one on [https://education.github.com/](https://education.github.com/git-cheat-sheet-education.pdf).

Enjoy! 

[git-cheat-sheet-education.pdf](https://github.com/community/community/files/12784800/git-cheat-sheet-education.pdf)

**Comment by queenofcorgis** [comment]
GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. Please note any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
